### PR TITLE
fixed level1 button position to bottom left and right

### PR DESCRIPTION
**Pull request by y1c1**

### CODE REVIEW

**Comment by y1c1** [comment]
A screenshot of how the level 1 page looks: (Not sure if this commit is necessary or not because we might have to change the icon and the style of font later)
Screenshot 2024-02-13 125110

**Comment by ashleyplusplus** [comment]
> A screenshot of how the level 1 page looks: (Not sure if this commit is necessary or not because we might have to change the icon and the style of font later) Screenshot 2024-02-13 125110

nah this is great!! love the sticky buttons, way easier to see how they show up here and then look at the code

File: level1.css [hunk 1]
```diff
@@ -6,8 +6,20 @@ h1 {
     color: rgb(128, 94, 156);
 }
 
-button {
+button1 {
```
Comment (ashleyplusplus) [inline level1.css]:
```suggestion
backbutton {
```
might be good to specify what the buttons do for readability

File: level1.css [hunk 2]
```diff
@@ -6,8 +6,20 @@ h1 {
...
     background-color: rgb(249, 221, 250);
     color: black;
     font-size: 36pt;
```
Comment (ashleyplusplus) [inline level1.css:12]: i'd like this to be a lil smaller but this can be edited later as well.

**Review by ashleyplusplus — COMMENTED** [review]
looks good, just a nitpick for readability and on font size

**Review by BethJ-20 — APPROVED** [review]

**Comment by Jsharp517** [comment]
Sweet

**Pull request merged**